### ISSUE TITLE
[Input Labels] Support for labels to the side of inputs 

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#5356]
# Platform

All

# Details

We don't currently support a way for a label to appear to the side of an input. This came up during accessibility work for the designer, where the Element Properties are built using adaptive card elements. Because we cannot put the label to the side of the input as is done in the designer, there is no way using standard adaptive card properties to correctly label the inputs for accessibility purposes. I worked around it for the purpose of fixing the designer bug, but we should support this type of layout for our customers as well, given that since we use it ourselves its clearly a useful scenario.

![image](https://user-images.githubusercontent.com/9091833/106928853-82453c80-66c8-11eb-97a7-a873a828ed8c.png)
